mi/dataset/parser/test/test_flort_dj_dcl.py

```diff
@@ -440,3 +423,20 @@ def test_ctdbp_cdef_dcl_flort_d(self):
         log.debug('===== END TEST CTDBPwFLORT:24 recs =====')
         log.debug('===== END TESTS ENHANCEMENT 9809  =====')
 
+    def test_bug_13245(self):
+        """
+        This test verifies that Issue #13245, where the re.match call in the parser
+        never returns, has been fixed.
+        """
+        log.debug('===== START TEST BUG 13245 =====')
+
+        with self.open_file('20141204.flort2.log') as in_file:
+            parser = FlortDjDclParser(self.rec_config, in_file, self.exception_callback)
+
+            particles = parser.get_records(5)
+            log.debug("Num particles: %d", len(particles))
+
+            self.assertEquals(len(particles), 0)
```

Comment:
There should be no particles from this file? Is that correct?

Reply:
Yes, that's correct.  This was the actual file that failed when parsing, which was the impetus of this ticket.

Reply:
So no records should be parsed from this file. Good.

Reply:
Correct.  It's suppose to raise an error and not get stuck.  As you can see below the exception count is 1.